compact-custom-header-editor.js

```diff
@@ -127,8 +127,9 @@ ${navigator.userAgent}
           margin-bottom: 0;
         }
         paper-button {
+          margin-top: 5px;
           background-color: var(--primary-color);
-          margin-left: 0;

```

Comment:
Without this the button is indented by 5px and the left hand side does not line up with any other element.